libs/@guardian/identity-auth/package.json

```diff
@@ -1,15 +1,18 @@
 {
 	"name": "@guardian/identity-auth",
-	"version": "4.0.0",
+	"version": "0.0.1",
```

Comment:
Personal preference, but I'd start versioning a library like this at `0.1.0`, since the first version defines the API rather than making minor patch changes, but the API may not be in a stable state yet, which would be `1.y.z`. Maybe CSTI has its own rules though! Cf.: https://stackoverflow.com/a/24154727